lms/models/application_instance.py

```diff
@@ -23,6 +23,9 @@ class ApplicationSettings(JSONSettings):
         JSONSetting("canvas", "folders_enabled", asbool),
         JSONSetting("canvas", "strict_section_membership", asbool),
         JSONSetting("canvas", "pages_enabled", asbool),
+        JSONSetting("canvas_studio", "client_id"),
+        JSONSetting("canvas_studio", "client_secret"),
```

Comment:
What is the rationale for encrypting individual secrets? Note that the entire database is already encrypted by RDS.

Reply:
It turns out I had a discussion with @seanh about this in 2018. The original external developers of Hypothesis decided to do this with Canvas secrets, and we've since copied the approach to other fields. See https://hypothes-is.slack.com/archives/CBN3DGW02/p1537874862000100.